datalad/interface/install_handle.py

```diff
@@ -69,14 +73,16 @@ def __call__(self, handle, path=None, name=None):
                              "make sense." % local_master.name)
 
         name_prefix = None
+        handle_name = None
         if parts[0] in local_master.git_get_remotes() \
                 and len(parts) >= 2:
             # 'handle' starts with a name of a known collection, followed by at
             # least a second part, separated by '/'.
             # Therefore assuming it's a handle's key, not an url
 
+            handle_name = handle[len(parts[0])+1:]
```

Comment:
I guess... Just that this piece craved for it :-)

On October 6, 2015 9:32:55 AM EDT, bpoldrack <notifications@github.com> wrote:
>>          if parts[0] in local_master.git_get_remotes() \
>>                  and len(parts) >= 2:
>>              # 'handle' starts with a name of a known collection,
>followed by at
>>              # least a second part, separated by '/'.
>>              # Therefore assuming it's a handle's key, not an url
>>  
>> +            handle_name = handle[len(parts[0])+1:]
>
>Right, but I would like to these things in a refactoring PR aiming to
>centralize common needs of the command implementations. Agree?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/datalad/datalad/pull/248/files#r41263358